tests/extract/test_incremental.py

```diff
@@ -192,17 +193,27 @@ def test_optional_incremental_from_config() -> None:
 
 @configspec
 class SomeDataOverrideConfiguration:
-    created_at: dlt.sources.incremental = dlt.sources.incremental('created_at', initial_value='2022-02-03T00:00:00Z')
+    created_at: ClassVar[dlt.sources.incremental] = dlt.sources.incremental('created_at', initial_value='2022-02-03T00:00:00Z')
 
 
 # provide what to inject via spec. the spec contain the default
 @dlt.resource(spec=SomeDataOverrideConfiguration)
 def some_data_override_config(created_at: dlt.sources.incremental = dlt.config.value):
     assert created_at.cursor_path == 'created_at'
-    assert created_at.initial_value == '2000-02-03T00:00:00Z'
```

Comment:
I changed this, I think this test was not correct before, was it? anyway it started failing